techniques/html/H62.html

```diff
@@ -2,115 +2,74 @@
-            explains the scope of the technique, and the presence of techniques for a specific
-            technology does not imply that the technology can be used in all situations to create
-            content that meets WCAG 2.1.
+         <h2>達成方法に関する重要な情報</h2>
+         <p>これらの達成方法 (参考) の使用法及び、それらが WCAG 2.1 達成基準 (規定) とどのように関係するかに関する重要な情報については、<a href="https://waic.jp/docs/WCAG21/Understanding/understanding-techniques">WCAG 達成基準の達成方法を理解する</a>を参照のこと。適用 (対象) セクションは、その達成方法の範囲について説明しており、特定の技術に関する達成方法があるからといって、WCAG 2.1 を満たすコンテンツを作成する際に、常にその技術が使用可能であるわけではない。
          </p>
       </section>
       <main>
          <section id="applicability">
-            <h2>Applicability</h2>
-            <p>XHTML 1.1 and HTML5</p>
-            <p>This technique relates to <span><a href="https://waic.jp/docs/WCAG21/Understanding/pronunciation">Success Criterion 3.1.6: Pronunciation</a> (Sufficient)</span>.
+            <h2>適用 (対象)</h2>
+            <p>XHTML 1.1 及び HTML5。</p>
+            <div class="note">
+               <div role="heading" class="note-title marker" aria-level="3">訳注</div>
+               <div>   
+                  <p>XHTML 1.1 は Superseded Recommendation としてマークされ、廃止された仕様である。
+                  </p>  
+               </div>
+            </div>
+            <p>これは<span><a href="https://waic.jp/docs/WCAG21/Understanding/pronunciation">達成基準 3.1.6: 発音</a> (十分な達成方法)</span> に関する達成方法である。
             </p>
          </section>
          <section id="description">
-            <h2>Description</h2>
-            <p>The objective of this technique is to use ruby annotation to provide information about
-               the pronunciation and meaning of a run of text where meaning is determined by
-               pronunciation.
+            <h2>解説</h2>
+            <p>この達成方法の目的は、読み方の情報、及び意味が読み方によって決まる場合の連続するテキストの意味に関する情報を提供するために、ルビを振ることである。
             </p>
-            <p>There are many languages in which a run of text may mean different things depending
-               on
-               how the text is pronounced. This is common in East Asian languages as well as Hebrew,
-               Arabic, and other languages; it also occurs in English and other Western European
-               languages. 
+            <p>多くの言語では、連続するテキストが読み方によって意味が異なる場合がある。これは、ヘブライ語、アラビア語、ほかの諸言語と同様に東アジア言語によくある。また、英語など西ヨーロッパ言語でも起こる。
             </p>
-            <p>Ruby Annotation allows the author to annotate a "base text," providing a guide to
-               pronunciation and, in some cases, a definition as well. Ruby is commonly used for
-               text
-               in Japanese and other East Asian languages. Ruby Annotation is defined as a module
-               for
-               XHTML 1.1 or HTML5. 
+            <p>「Ruby Annotation」は XHTML 1.1 又は HTML5 のモジュールのひとつとして定義されている。コンテンツ制作者は「Ruby Annotation」によって、読み方や、場合によっては定義を提供すべく、「ベーステキスト」に注釈を加えることができる。日本語など東アジア言語のテキストでは、ルビが当たり前に利用される。
```

Comment:
分の順序が入れ替わっているので原文にあわせました
```suggestion
            <p>コンテンツ制作者はRuby Annotationによって、読み方や、場合によっては定義を提供すべく、「ベーステキスト」に注釈を加えることができる。日本語など東アジア言語のテキストでは、ルビが当たり前に利用される。Ruby Annotationは XHTML 1.1 又は HTML5 のモジュールのひとつとして定義されている。
```